mmv1/products/compute/terraform.yaml

```diff
@@ -2214,6 +2221,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         description: |
           {{description}}
           If it is not provided, the provider region is used.
+      enable: !ruby/object:Overrides::Terraform::PropertyOverride
+        custom_expand: 'templates/terraform/custom_expand/bool_to_upper_string.erb'
```

Comment:
The API attribute is actually an enum supporting only "TRUE" and "FALSE" hence this hack here. This has been discussed in the initial PR this one is based on and we [agreed these was the best way to proceed](https://github.com/GoogleCloudPlatform/magic-modules/pull/3258#discussion_r393939239).